adb_graphics/figures/skewt.py

```diff
@@ -455,21 +521,29 @@ def _title(self):
         vtime = self.date_to_str(self.valid_dt)
 
         # Top Left
-        plt.title(f"Analysis: {atime}\nFcst Hr: {self.fhr}",
-                  fontsize=16,
-                  loc='left',
-                  position=(0, 1.03),
-                  )
+#        plt.title(f"Analysis: {atime}\nFcst Hr: {self.fhr}",
+#                  fontsize=16,
+#                  loc='left',
+#                  position=(0, 1.03),
+#                  )
+        plt.suptitle(f"Analysis: {atime}\nFcst Hr: {self.fhr}",
+                     fontsize=16,
+                     ha='right')
```

Comment:
It seems that[ suptitle ](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.suptitle.html) has separate x, y arguments (rather than `position` like the `plt.title`. Could you try those to get the title to behave like the pre-existing title elements?

This is also a pretty nice thread on [how to position the suptitles](https://stackoverflow.com/questions/55767312/how-to-position-suptitle).

Reply:
I went back to plt.title and adjusted the position arguments.